### DISCUSSION TITLE
Android and iOS apps

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#2193] [Anything Else]
"do you have any plans for an iOS app in the future? I have multiple clients (I made them use your solution) on Plausible, and I don't have an easy way to switch between accounts on the go. An iOS app would solve this problem easily"

([feedback via twitter](https://twitter.com/Suganthanmn/status/1567827909775163394))

**Comment by muesliq** [comment]
Alternative: Account switcher in top right menu. (Like Twitter or Instagram.)

**Reply by metmarkosaric** [reply]
it's also possible to have your clients invite you as a team member to their sites so you'll be able to see them all (and site settings too in case you get the admin role) directly from your own account. see https://plausible.io/docs/users-roles

**Comment by oliveratgithub** [comment]
Maybe make plausible.io a web app instead of full blown native mobile app?

And one small thing: would be nice to have a login + redirect, instead of showing a 404 error when accessing a property deeplink - without being logged in. Would help to get quicker to the dashboard as long as there is only the web view; including a iOS Home Screen shortcut.

**Reply by Swiss-Mac-User** [reply]
Looks like better PWA support is coming soon: #2405

**Comment by metmarkosaric** [comment]
we have listed some community-made iOS apps and widgets here: https://plausible.io/docs/integration-guides#ios-apps-and-widgets-for-quick-overview-of-your-stats

**Comment by mertbulan** [comment]
I developed an iOS app that would help you to keep an eye on your Plausible Analytics data on your iPhone: http://charsible.app 

It has multiple widgets that you can use not only on iOS but also on iPadOS and macOS. Let me know if you have any questions or suggestions for the app :)

**Reply by tiste** [reply]
@mertbulan can we have life license? instead of another subcription?

**Reply by mertbulan** [reply]
@tiste thanks for the feedback! I will think about this option.

**Reply by mertbulan** [reply]
@tiste I added lifetime access also reduced the subscription prices significantly.

**Comment by rivafarabi** [comment]
A bit late to the conversation, but I built [Applausible](https://applausible.app/) last year. It’s available on Android and iOS, with home widgets and a tablet view support.

**Comment by jorijn** [comment]
Hey everyone. I've been following this discussion and wanted to share that I built a native iOS companion for Plausible. It was just released on the App Store: [Arbory](https://apps.apple.com/app/id6760103214)

It connects to both Plausible Cloud and self-hosted instances via API key.

Features:
- Full dashboard with all 9 sections and interactive charts
- Home screen widgets (small, medium, large) and lock screen widgets
- Traffic spike/drop push notifications and daily summary digests

Privacy is important: no servers inbetween, your phone connects to Plausible directly. The free version is fully usable, it offers the full dashboard & filtering experience for one site. Additional features can be unlocked with a one-time payment, no subscriptions.
